superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -849,17 +849,26 @@ export default function VizTypeGallery(props: VizTypeGalleryProps) {
                 grid-area: examples-header;
               `}
             >
-              {!!selectedVizMetadata?.exampleGallery?.length && t('Examples')}
+              {t('Examples')}
             </SectionTitle>
             <Examples>
-              {(selectedVizMetadata?.exampleGallery || []).map(example => (
+              {selectedVizMetadata?.exampleGallery.length ? (
```

Comment:
Maybe this looks more readable?

```
<Examples>
  {(
    selectedVizMetadata?.exampleGallery || [
      {
        url: selectedVizMetadata?.thumbnail,
        caption: selectedVizMetadata?.thumbnail,
      },
    ]
  ).map(example => (
    <img
      key={example.url}
      src={example.url}
      alt={example.caption}
      title={example.caption}
    />
  ))}
</Examples>
```

Reply:
Hi @michael-s-molina, The above piece of code which you have mentioned doesn't render the thumbnail image in case of no exampleGallery.

But,
```jsx
           <Examples>
              {(selectedVizMetadata?.exampleGallery?.length
                ? selectedVizMetadata.exampleGallery
                : [
                    {
                      url: selectedVizMetadata?.thumbnail,
                      caption: selectedVizMetadata?.name,
                    },
                  ]
              ).map(example => (
                <img
                  key={example.url}
                  src={example.url}
                  alt={example.caption}
                  title={example.caption}
                />
              ))}
            </Examples>
```
this code works.

Can I go and use this piece of code and modify this PR?

Reply:
👍🏼